src/components/ColorPicker/ColorPicker.tsx

```diff
@@ -42,7 +42,6 @@ export default class ColorPicker extends React.PureComponent<Props, State> {
       return;
     }
 
-    // eslint-disable-next-line react/no-did-mount-set-state
```

Comment:
calling setState in componentDidMount shouldn't happen that often but it is a legitimate use according to the react docs. See https://github.com/yannickcr/eslint-plugin-react/issues/1754 for context.